backend/fleet_management/templates/new_drive_email.html

```diff
@@ -0,0 +1,28 @@
+Hello {{ passenger.first_name }}!
```

Comment:
How about adding a locale field to passenger and basing email content based on that?

Reply:
I thought about it but I have no strong opinion about that. @danutaj23 What do you think about this idea?

Reply:
I am imagining being a user of the application who speaks primarily polish. I'd need to scroll to the bottom of the email on my phone each time when I get it and I'd get discouraged easily.